string class_label
float32 x
float32 y